### PR TITLE
Segemehl030

### PR DESCRIPTION
**Pull request by jfallmann**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Updated segemehl 0.2 and added segemehl 0.3

### CODE REVIEW

File: recipes/segemehl/meta.yaml [hunk 1]
```diff
@@ -1,33 +1,46 @@
...
   name: segemehl
-  version: 0.2.0
+  version: {{ version }}
 
 about:
   home: http://www.bioinf.uni-leipzig.de/Software/segemehl/
   summary: Short read mapping with gaps
 build:
-  number: 4
+  number: 1
```
Comment (bgruening) [inline recipes/segemehl/meta.yaml]: 0 for new packages

File: recipes/segemehl/meta.yaml [hunk 2]
```diff
@@ -1,33 +1,46 @@
...
     - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
   host:
     - zlib
     - ncurses
+    - htslib
+#    - libcxx
+#    - libgcc
   run:
     - zlib
```
Comment (bgruening) [inline recipes/segemehl/meta.yaml]: zlib can also be removed as run dependency

**Comment by jfallmann** [comment]
@bgruening ready to merge when you are ;)

**Comment by bgruening** [comment]
@jfallmann the OSX test is not running. Can you please accept the invitation which I send you? The OSX stuff is running on your private circle-ci account. This is strange.

**Comment by jfallmann** [comment]
@bgruening can you restart the checks, i unsubscribed my private fork, should work now

**Pull request closed without merging**